samples/ControlCatalog/Pages/ButtonPage.xaml

```diff
@@ -20,6 +20,10 @@
               <Style.Resources>
                 <SolidColorBrush x:Key="ThemeBorderMidBrush">Red</SolidColorBrush>
                 <SolidColorBrush x:Key="ThemeControlHighBrush">DarkRed</SolidColorBrush>
+                <SolidColorBrush x:Key="ButtonBorderBrush">Red</SolidColorBrush>
+                <SolidColorBrush x:Key="ButtonBackground">DarkRed</SolidColorBrush>
+                <SolidColorBrush x:Key="ButtonBackgroundPointerOver">Red</SolidColorBrush>
+                <SolidColorBrush x:Key="ButtonBackgroundPressed">OrangeRed</SolidColorBrush>
```

Comment:
Not related to this PR, but I added those lines just to complete ButtonsPage for Fluent theme. "Re-styled" button now actually re-styled :)